docs/controls/BubbleSingleDataset.md

```diff
@@ -0,0 +1,327 @@
+        -   [Constraints](#constraints)
+
+## Usage
+
+### Structure
+
+You will **not** need all the properties in the example below.
+Check out _optional_/_required_ properties explained in the [JSON Properties](#json-properties) section.
+
+```javascript
+var root = {
+    bindTo: "#root",
+    axis: {
+        x: {
+            type: Carbon.helpers.AXIS_TYPE.TIME_SERIES,
+            label: "Some X Label",
+            lowerLimit: "2016-01-01T12:00:00Z",
+            upperLimit: "2017-01-01T12:00:00Z"
+        },
+        y: {
+            label: "Some Y Label",
+            lowerLimit: 0,
+            upperLimit: 20
+        }
+    }
+};
+
+var data = {
+    key: "uid_1",
+    label: {
+        display: "Data Label 1"
+    },
+    color: Carbon.helpers.COLORS.BLUE,
+    values: [
+        {
+            x: "2016-02-03T12:00:00Z",
+            y: 4
+        },
+        {
+            x: "2016-05-01T12:00:00Z",
+            y: 15
+        },
+        {
+            x: "2016-10-01T12:00:00Z",
+            y: 10
+        }
+    ]
+};
+var bubbleDefault = Carbon.api.graph(root);
+bubbleDefault.loadContent(Carbon.api.bubble(data));
+```
+
+#### Weight Based Bubble
+
+For weight based bubble Graph, provide weight with min and max and also pass weight as third paramter in the values and the weight(third parameter) should be in the range of min and max provide in the weight.
+
+```javascript
+
+var dataWeightBased = {
+    key: "uid_1",
+    label: {
+        display: "Data Label 1"
+    },
+    weight: {
+        min: 10,
+        max: 50
+    }
+    color: Carbon.helpers.COLORS.BLUE,
+    values: [
+        {
+            x: "2016-02-03T12:00:00Z",
+            y: 4,
+            weight: 15
+        },
+        {
+            x: "2016-05-01T12:00:00Z",
+            y: 15,
+            weight: 20
+        },
+        {
+            x: "2016-10-01T12:00:00Z",
+            y: 10,
+            weight: 40
+        }
+    ]
+};
+var bubbleWeight = Carbon.api.graph(root);
+bubbleWeight.loadContent(Carbon.api.bubbleSingleDataset(dataWeightBased));
+```
+
+#### Color Based Bubble
+
+For a Color Based Bubble Graph, provide `palette` and load the following content:
+
+```javascript
+var colorBasedData = {
+    key: "uid_2",
+    label: {
+        display: "Data Label 2"
+    },
+    palette: BUBBLE.PALETTE.ORANGE,
+    values: [
+        {
+            x: "2016-03-01T12:00:00Z",
+            y: 14
+        },
+        {
+            x: "2016-04-10T12:00:00Z",
+            y: 1
+        },
+        {
+            x: "2016-11-01T12:00:00Z",
+            y: 18
+        }
+    ]
+};
+bubbleColorBased = Carbon.api.graph(root);
+bubbleColorBased.loadContent(Carbon.api.bubbleSingleDataset(colorBasedData));
+```
+
+#### Weight Color Based Bubble
+
+For a Weight and Color Based Bubble Graph, provide hue with lowerShade and upperShade, also provide weight with min and max also pass weight(thrid parameter)in the values as below content:
```

Comment:
Are we still using hue?

Reply:
fixed: 
https://github.com/cerner/carbon-graphs/pull/308/commits/1a920bb9dc6643049ba404be1a3bcc8cd672e8ac